package/linux-firmware/linux-firmware.mk

```diff
@@ -72,6 +72,18 @@ LINUX_FIRMWARE_FILES += intel/ibt-*
 LINUX_FIRMWARE_ALL_LICENSE_FILES += LICENCE.ibt_firmware
 endif
 
+# Mediatek MT7921 Bluetooth
+ifeq ($(BR2_PACKAGE_LINUX_FIRMWARE_MT7921_BT),y)
+LINUX_FIRMWARE_FILES += mediatek/BT_RAM_CODE_MT7961_1_2_hdr.bin
```

Comment:
Is MT7921 and MT7961 the same?

Reply:
Hm, I see this comes from `drivers/net/wireless/mediatek/mt76/mt792x.h`. Seems correct :+1:

Reply:
Yeah, I haven't searched for details but it's indeed correct :shrug: